workshops/check.py

```diff
@@ -314,6 +357,16 @@ def check_validity(data, function, errors, error_msg):
     return valid
 
 
+def check_blank_lines(raw_data, errors, error_msg):
+    '''Check for blank line in category headers.'''
```

Comment:
'''Blank lines are not allowed in category headers.''' <<== will be clearer to non-native English speakers.